source/backend/api/Areas/Leases/Controllers/LeaseController.cs

```diff
@@ -53,11 +57,27 @@ public LeaseController(ILeaseRepository leaseRepository, IMapper mapper)
         [SwaggerOperation(Tags = new[] { "lease" })]
         public IActionResult GetLease(int id)
         {
-            var lease = _leaseRepository.Get(id);
+            var lease = _leaseService.GetById(id);
             var mapped = _mapper.Map<Models.Lease.LeaseModel>(lease);
             return new JsonResult(mapped);
         }
 
+        /// <summary>
+        /// Get the lease for the specified primary key 'id'.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("concept/{id:long}")]
+        [HasPermission(Permissions.LeaseView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(IEnumerable<Api.Models.Concepts.LeaseModel>), 200)]
```

Comment:
nit: I believe the return type shouldn't be an **IEnumerable** if we only return one lease?